web/src/main/java/se/inera/intyg/intygstjanst/web/service/dto/citizen/CitizenCertificateRecipientDTO.java

```diff
@@ -0,0 +1,12 @@
+package se.inera.intyg.intygstjanst.web.service.dto.citizen;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class CitizenCertificateRecipientDTO {
+    String id;
+    String name;
+    String sent;
```

Comment:
This should be a LocalDateTime and we let the serialisation transform it to a String when part of a response in a rest call.